src/ray/gcs/gcs_server/gcs_server.h

```diff
@@ -48,6 +48,8 @@ struct GcsServerConfig {
   bool grpc_based_resource_broadcast = false;
   bool grpc_pubsub_enabled = false;
   std::string log_dir;
+  // This includes the config list of raylet.
```

Comment:
Side note: it seems `config_list` severely lacks documentation upstream and downstream, and its purpose (looks like default raylet config) is not intuitive from its name. Hopefully it can be cleaned up in future.

Reply:
Yes, agreed. I'm not sure what's this :( I only try to make the logic the same as before.